### PR TITLE
use constants in generator-common and cypress

### PR DESCRIPTION
**Pull request by Tcharl**
Contributes to [14235](https://github.com/jhipster/generator-jhipster/issues/14235)

---

Please make sure the below checklist is followed for Pull Requests.

- [ ] [All continuous integration tests](https://github.com/jhipster/generator-jhipster/actions) are green
- [x] Tests are added where necessary
- [x] [jhipster-online](https://github.com/jhipster/jhipster-online) was updated if necessary
- [x] Documentation is added/updated where necessary
- [x] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/main/CONTRIBUTING.md) are followed

When you are still working on the PR, consider converting it to Draft (bellow reviewers) and adding `skip-ci` label, you can still see CI build result at your branch.

### CODE REVIEW

**Review by DanielFran — CHANGES_REQUESTED** [review]

**Comment by Tcharl** [comment]
@DanielFran thank you for the review and remark. Good to merge now?

File: generators/database-changelog-liquibase/index.js [hunk 1]
```diff
@@ -42,6 +48,23 @@ module.exports = class extends BaseGenerator {
...
+        this.loadAppConfig();
+        this.loadClientConfig();
+        this.loadServerConfig();
+        this.loadTranslationConfig();
+      },
+    };
+  }
+
+  get loading() {
+    if (useBlueprints) return;
```
Comment (mshima) [inline generators/database-changelog-liquibase/index.js:66]: Blueprint support is not complete need to add `if (useBlueprints) return;` to others priorities.

Comment (Tcharl) [reply generators/database-changelog-liquibase/index.js:66]: Thanks, forgot about this one: I'll need a crash course on the impact between extending basegenerator and and blueprintbasegen: I do not really understand the impacts (I only know that there are some more steps in the lifecycle)

**Review by mshima — APPROVED** [review]
LGTM

**Pull request merged**